### ISSUE TITLE
Allow to write metadata

### ISSUE THREAD

**Issue by tcitworld** [gino0631/nextcloud-metadata#5]
The full getID3 library seems to support this.

**Comment by ChildishGiant** [comment]
This would be really useful for fixing incorrect data without having to have the owner of the server go into the command line. Any updates on this?

**Comment by gino0631** [comment]
@ChildishGiant thanks for your comment. What types of files you would like to be able to update metadata for?

**Comment by ChildishGiant** [comment]
![image](https://user-images.githubusercontent.com/13716824/82855323-e8a37980-9f02-11ea-8eb7-be9f172f76ca.png)
For me it's mainly mp3 metadata so I can easily handle my music library

**Comment by gino0631** [comment]
OK, for mp3 files that should be doable.

**Comment by RomainTT** [comment]
Hey ! Any updates about this ? I am also looking for a tool to edit music metadata directly in Nextcloud. Thanks for your work !

**Comment by FabianVolkers** [comment]
I would love to see this with image metadata as well, I want to supply additional location data for image files that are missing it.

**Comment by gino0631** [comment]
@RomainTT no news so far, sorry.
@FabianVolkers did you mean setting or editing GPS coordinates recorded in the image files?

**Comment by FabianVolkers** [comment]
@gino0631 Yes exactly, mostly setting GPS coordinates where they do not exist. I would assume it's the same logic as editing where they do exist.

**Comment by gino0631** [comment]
@FabianVolkers OK. I assume setting the coordinates using a map, and also a possibility to search for a location when it is not set yet? Any other thoughts about the possible implementation?

**Comment by Gymnae** [comment]
I'd love to be able to edit EXIF data, for adding captions and descriptions

**Comment by whitemanthedj** [comment]
I know it's been a while, but I have been searching for a way to add EXIF data to a photo of older photos that didn't have the location stored already (ideally since the metadata tab can already open the location of existing EXIF data in the map, allowing the user to select a point on the map would be super useful rather than figuring out the GPS coordinates themselves). So plus 1 to that being requested :)

**Comment by gino0631** [comment]
@whitemanthedj thanks for the comment 👍

**Comment by benoit-smith** [comment]
I, too, would love to be able to fix the file metadata in my music library (mostly FLAC) through this addon.

**Comment by Deuracell** [comment]
I also want this! 
Would be especially nice to be able to add Nextcloud tags to a file and having them be written to the meta data. That why tags are preserved outside of Nextcloud.

**Comment by justinlimbo** [comment]
I would also like to edit the id3-tags of my mp3 files. Any updates on this?
